kore/src/Kore/Step/Simplification/Equals.hs

```diff
@@ -58,8 +58,7 @@ import Kore.Step.Simplification.AndTerms
     ( maybeTermEquals
     )
 import {-# SOURCE #-} qualified Kore.Step.Simplification.Ceil as Ceil
-    ( makeEvaluate
-    , makeEvaluateTerm
+    ( makeEvaluateTerm
```

Comment:
Why is this still needed?

Reply:
It's not either, and if I also remove the import from `Kore.Builtin.Builtin`, then we can remove the `.hs-boot` file for `Ceil`! :tada: